deploy/finicky-whiskers.nomad

```diff
@@ -108,8 +108,8 @@ job "finicky-whiskers" {
       // `spin up` but then settles back down with small CPU spikes
       // during gameplay. Mem appears to stay at around 600-700MB.
       resources {
-        cpu = 1000
-        memory = 800
+        cpu = 2000
```

Comment:
Strictly speaking, this doesn't need to be bumped... but CPU does spike quite high on startup (upwards of 4k), which is allowed/fine.  However, we have plenty of CPU to spare on the VMs so thought it'd be a good idea to reserve a bit more.